### ISSUE TITLE
Publish the "common" library as a separate project with a common name

### ISSUE THREAD

**Issue by MurzNN** [bjw-s/helm-charts#239]
# Details

Your "common" Helm library is really amazing! It simplifies the things a lot for deploying most of the general applications!

But it's really not so visible for others as a sub-part of your whole project "bjw-s/helm-charts".

So, I want to advise you to publish it as a separate project, with a more general name like "helm-common-app-library", with several easy and short examples of how to use it on the main Readme file, showing all the benefits of it, comparing to a classic approach.

This should make it more visible and searchable for newbies!

What do you think about this idea?

**Comment by MurzNN** [comment]
Also, it should involve more new contributors, because the current Github repo looks like your personal project, not a general project, that can be maintained by the community.

**Comment by reitermarkus** [comment]
@MurzNN, note that this was actually previously part of @k8s-at-home, which was deprecated because it was too much maintenance effort for only a few maintainers, see https://github.com/k8s-at-home/charts/issues/1761.

So

> maintained by the community

is easier said than done.

This is just for context, not an argument against your suggestions.

**Comment by MurzNN** [comment]
Yeah, @k8s-at-home was a great project! But supporting too many charts by a few developers is a pretty tough task.

Now you concentrated on the common library, which allows other developers to build their own chats, based on this great library! So, I guess, the support volume from your person will stay the same, but the popularity will grow.

**Comment by bjw-s** [comment]
Apologies for not responding to this sooner. While I appreciate the sentiment, I am quite happy with where the project is at currently. 

Folks that use it know where to find it and how to contribute if they want. I also don't really have the ambition to turn it into a "faster moving" project to be honest, so _for now_ I am not really planning on changing anything in that regard.
